demos/hazard/ScenarioCase3/job.ini

```diff
@@ -0,0 +1,27 @@
+[general]
+
+description = Scenario Calculation with Complex Fault Rupture
```

Comment:
The description should be changed to Point Rupture

Reply:
Actually in current master the description says `Scenario Calculation with rupture_dict` so it was already changed.